src/custom/components/AccountDetails/styled.ts

```diff
@@ -262,8 +262,10 @@ export const LowerSection = styled.div`
   }
 `
 
+// TODO: Prevent beyond 3 level selector nestings
```

Comment:
Noticed a deep nesting of selectors. In general I'd opt for max 2, at most 3 levels deep. Otherwise refactor the selector logic to make the general code more readable.